### PR TITLE
Add leitnerbox extension

### PR DESCRIPTION
**Pull request by GZyannick**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/leitnerbox/package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "leitnerbox",
+  "title": "LeitnerBox",
+  "description": "",
```
Comment (pernielsentikaer) [inline extensions/leitnerbox/package.json:5]: Missing description

File: extensions/leitnerbox/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# LeitnerBox
```
Comment (pernielsentikaer) [inline extensions/leitnerbox/README.md:1]: Maybe write a few lines about what it is

File: extensions/leitnerbox/package.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  "author": "gzyannick",
+  "categories": [
+    "Applications",
+    "Fun"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "create-a-question",
+      "title": "Create a Question",
```
Comment (pernielsentikaer) [inline extensions/leitnerbox/package.json:16]:
```suggestion
      "title": "Create a Question",
      "subtitle": "LeitnerBox",
```

File: extensions/leitnerbox/package.json [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+      "description": "Create a question for your leitner box",
+      "mode": "view"
+    },
+    {
+      "name": "daily-question",
+      "title": "Daily Question",
```
Comment (pernielsentikaer) [inline extensions/leitnerbox/package.json:22]:
```suggestion
      "title": "Daily Question",
      "subtitle": "LeitnerBox",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- You're missing an extension icon, you can generate [one here](https://icon.ray.so/) 

- Could you add at least one [screenshot](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) of the extension

- Missing a success toast after adding a question

- Seems like it's possible to submit the form without anything in it, you can use [useForm](https://developers.raycast.com/utilities/react-hooks/useform) to help with the validation

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- You're missing an extension icon, you can generate [one here](https://icon.ray.so/) 

- Could you add at least one [screenshot](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) of the extension

- Missing a success toast after adding a question

- Seems like it's possible to submit the form without anything in it, you can use [useForm](https://developers.raycast.com/utilities/react-hooks/useform) to help with the validation

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by GZyannick** [comment]
thanks you for you feedback i will do the description logo and toast

**Comment by pernielsentikaer** [comment]
Let me know if you need any help 🙂

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
I'm not sure if you're done yet?

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by pernielsentikaer** [comment]
@GZyannick is this ready for another review?

**Comment by GZyannick** [comment]
no i was just sending my pull request sorry for the notification

**Comment by pernielsentikaer** [comment]
I just checked the extension out fresh and added two questions, when should I be able to see it in `Daily Questions`

leitnerbox 2024-02-21 at 13 41 37

**Comment by GZyannick** [comment]
the system work like boxes when you add a question it will come 1 day after (box 1) there is 5box 
{
   box1: 1day,
   box2: 2days,
   box3: 4days,
   box4: 8days,
   box5: 10days
 }
 
 the goal is to learn something with consistency and time to memorize it in the long term memory

**Comment by GZyannick** [comment]
if you prefere for the ux i can add something like:

 You have no question for today
 2 question in 1 day 
 1 question in 2days
 6 question in 5 days...

**Comment by pernielsentikaer** [comment]
Can we add some more information so people is not getting confused about it?

**Comment by GZyannick** [comment]
Didnt close the pull request 

i have an issue to npm run publish i have in extensions on slack 
i will tell you when i can push the update

**Pull request closed without merging**